pages/content/amp-dev/documentation/guides-and-tutorials/develop/animations/triggering_css_animations.md

```diff
@@ -73,11 +143,77 @@ The `toggleClass` action can apply to other elements as well and toggle between
 
 If you need to remove a class and disallow reapplication, add the `force` attribute with a value of `false`. If you need to addd a class and disallow removal, add `force` with a value of `true`.
 
-## Animate with Numerous Classes
+## Animate with numerous classes
```

Comment:
I'm not a fan of the title, can we replace numerous classes with something that better explains what's being demonstrated?